tests/modules/msk/msisensor-ct/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: msisensor-ct test_msisensor_ct
+  command: nextflow run ./tests/modules/msk/msisensor-ct -entry test_msisensor_ct -c ./tests/config/nextflow.config -c ./tests/modules/msk/msisensor-ct/nextflow.config -stub-run
```

Comment:
This is the command to actually run the stubtest. You can run this in your terminal to see if everything is working. Don't worry about the pytest for right now. Feel free to reach out if get an error.

If you want to run your 'real data test', you can change the `-entry` to another test name and remove `-stub-run`.